React 18 support

Will you support react 18, and do you have any idea what changes need to be made to support this? 👍 
Happy to help if I got some pointers on what to do.

Release notes for 6.7.4 indicate that react 18 should be supported. However I still get:
npm ERR! Could not resolve dependency:
npm ERR! peer react@"16.* || 17.*" from @projectstorm/react-canvas-core@6.7.4
npm ERR! node_modules/@projectstorm/react-canvas-core
npm ERR!   @projectstorm/react-canvas-core@"^6.7.4" from the root project

> Release notes for 6.7.4 indicate that react 18 should be supported. However I still get: npm ERR! Could not resolve dependency: npm ERR! peer react@"16.* || 17.*" from @projectstorm/react-canvas-core@6.7.4 npm ERR! node_modules/@projectstorm/react-canvas-core npm ERR! @projectstorm/react-canvas-core@"^6.7.4" from the root project

I have the same problem, I execute --force and --legacy-peer-deps, but not work, how to solve this problem?

--- a/packages/react-canvas-core/package.json
+++ b/packages/react-canvas-core/package.json
@@ -33,7 +33,7 @@
 	},
 	"peerDependencies": {
 		"lodash": "4.*",
-		"react": "16.* || 17.*"
+		"react": "18.*"
 	},
 	"gitHead": "bb878657ba0c2f81764f32901fd96158a0f8352e"
 }

